source/documentation/deploying_services/s3.md

```diff
@@ -106,6 +110,51 @@ An example `VCAP_SERVICES` environment variable:
 ```
 You must pass credentials from your `VCAP_SERVICES` environment variable to your AWS software development kit (SDK). Use the appropriate [Amazon SDK documentation](https://aws.amazon.com/developer/tools/#sdks) to do this.  
 
+### Connect to an S3 bucket from outside of the GOV.UK PaaS
+
+To connect to the S3 bucket from outside of the GOV.UK PaaS, you must create a service key and credentials that allow external access to the bucket.
+
+1. Run the following to create a service key and credentials that allows external access:
+
+    ```
+    cf create-service-key SERVICE_NAME SERVICE-KEY -c '{"allow_external_access": true}'
+    ```
+    where:
+    - `SERVICE_NAME` is a unique descriptive name for the S3 bucket
+    - `SERVICE-KEY` is a unique descriptive name for the service key
+
+1. Run the following to get the external access service key credentials:
+
+    ```
+    cf service-key SERVICE_NAME SERVICE-KEY
+    ```
+
+    You will see output similar to the following:
+
+    ```
+    {
+     "aws_access_key_id": "AWS_ACCESS_KEY",
+     "aws_region": "eu-west-2",
+     "aws_secret_access_key": "AWS_SECRET_ACCESS_KEY",
+     "bucket_name": "BUCKET_NAME_GUID",
+     "deploy_env": ""
+    }
+    ```
+
+    Use these credentials to connect to the S3 bucket from outside of the GOV.UK PaaS.
```

Comment:
The only thing here is that I wonder if we should point out that the region specific **must** be used here, as we are specifically creating the buckets in the same region as the PaaS.

I'll need to do some digging to find out what that actually is though

Reply:
not a valid concern: `aws s3 ls s3://BUCKET_NAME/` works A-OK